DOC-8178 FGAC goes GA (#17521)
cockroachdb/docs: src/current/cockroachcloud/managing-access.md

---

> title: Managing Users, Roles, and Service Accounts in CockroachDB Cloud

Reviewer: This title change was not showing up on the sidebar... they mismatch, perhaps that is on purpose?

## Manage service accounts

> The access management model for service accounts is unified with that for users, meaning service accounts may have all of the same [access roles](authorization.html#organization-user-roles), although it should be noted that service accounts and users still differ in the actions they can performed, as only users can access the console, and only service accounts can access the API, and the console and API differ in functionality.

Reviewer: What does "unified" mean here? Like mirrored? Not sure on that, but suggest breaking this sentence up a little anyway (have kept your unified language for now)
```suggestion
The access management model for service accounts is unified with the [user model](#manage-an-organizations-users). This means that service accounts may have all of the same [access roles](authorization.html#organization-user-roles). However, service accounts and users still differ in the actions they can perform: only users can access the console, and only service accounts can access the API. The console and API differ in functionality.
```

Legacy service accounts created prior to the current authorization model may still have the following legacy roles: (ADMIN, CREATE, EDIT, READ, DELETE). Refer to [Service accounts](authorization.html#service-accounts).

> It is recommended to update service accounts to fine-grained access roles, by [editing their roles](#edit-roles-on-a-service-account).

Reviewer 2: ```suggestion
It is recommended to update service accounts to roles in the new authorization model, by [editing their roles](#edit-roles-on-a-service-account).
```
